src/features/projects/components/ProjectSnippet.tsx

```diff
@@ -63,7 +63,12 @@ export default function ProjectSnippet({
 
   const { token } = useUserProps();
   const handleOpen = () => {
-    const url = getDonationUrl(project.slug, token, embed, callbackUrl);
+    const url = getDonationUrl(
+      project.slug,
+      token,
+      embed || 'false',
```

Comment:
Any reason you've kept the fallback for `embed` as 'false'? Why not undefined instead?